docs/source/developer/how_to_extend.rst

```diff
@@ -73,8 +75,16 @@ ensuring also that ``openwisp_radius`` has been removed:
         # 'myradius', <-- replace with your app-name here
         'openwisp_users',
         'private_storage',
+        'drf_yasg'
```

Comment:
this is not mentioned in https://openwisp-radius.readthedocs.io/en/latest/developer/setup.html, can you add it there as well?

Reply:
Done! :smile: